packages/node/src/methods/request-handler.ts

```diff
@@ -1,8 +1,10 @@
+import { InstructionExecutor } from "@counterfactual/machine";
 import {
-  InstructionExecutor,
-  types as machineTypes
-} from "@counterfactual/machine";
-import { Protocol } from "@counterfactual/machine/dist/src/types";
+  Context,
+  Opcode,
+  Protocol,
+  ProtocolMessage
+} from "@counterfactual/machine/dist/src/types";
```

Comment:
Wasn't the whole point of this PR to import all these things from "@counterfactual/machine"?

Reply:
Oh woops my bad, fixing